newIDE/app/src/AssetStore/ExampleStore/ExampleStoreContext.js

```diff
@@ -91,7 +91,7 @@ export const ExampleStoreStateProvider = ({
 
           console.info(
             `Loaded ${
-              exampleShortHeaders.length
+              exampleShortHeaders.length ? exampleShortHeaders.length : 0
```

Comment:
```suggestion
              exampleShortHeaders ? exampleShortHeaders.length : 0
```

or (but this won't prevent a crash if null/undefined)

```suggestion
              exampleShortHeaders.length || 0
```